recipe/meta.yaml

```diff
@@ -10,17 +10,18 @@ source:
   sha256: 46f0c74d73efa723776764b43c9739f68052495dd3d734319c1d0eb58511f15b
 
 build:
-  noarch: python
   number: 0
   script: {{ PYTHON }} -m pip install . -vv
+  skipt: true  # [py<36]
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
     - setuptools-scm
```

Comment:
The bounds could be added:
https://github.com/pyscaffold/configupdater/blob/v3.1.1/pyproject.toml#L3

Reply:
I added the bounds to `setuptools` and `setuptools-scm`